python/hail/dataset.py

```diff
@@ -1231,6 +1231,189 @@ def count(self):
+                max_parent_ab=0.05, min_child_ab=0.20, min_depth_ratio=0.10):
+        """Call de novo variation from trio data.
+
+        **Examples**
+
+        Call de novo events and export them to a file:
+
+        >>> pedigree = Pedigree.read('data/myStudy.fam')
+        >>> priors = hc.import_table('data/gnomadFreq.tsv', impute=True).key_by('Variant')
+        >>> denovo_kt = (vds.annotate_variants_table(priors, root='va.gnomAD')
+        ...                  .de_novo(pedigree, 'va.gnomAD'))
+        >>> denovo_kt.export('denovo_calls.tsv')
+
+        **Notes**
+
+        This method replicates the functionality of `Kaitlin Samocha's de novo caller <https://github.com/ksamocha/de_novo_scripts>`__.
+        The version corresponding to git commit ``bde3e40`` is implemented in Hail with her permission and assistance.
+
+        This method produces a :py:class:`.KeyTable` with the following columns:
+
+            - **variant** (*Variant*) -- Variant at which parents are homozygous reference and proband is heterozygous.
+            - **probandID** (*String*) -- Sample ID of proband.
+            - **fatherID** (*String*) -- Sample ID of father.
+            - **motherID** (*String*) -- Sample ID of mother.
+            - **isFemale** (*Boolean*) -- Sex of proband, true if female.
+            - **confidence** (*String*) -- Validation likelihood of event. One of: 'HIGH', 'MEDIUM', 'LOW'
+            - **probandGt** (*Genotype*) -- Genotype of the proband
+            - **fatherGt** (*Genotype*) -- Genotype of the father
+            - **motherGt** (*Genotype*) -- Genotype of the mother
+            - **pDeNovo** (*Float64*) -- Posterior probability that the event is a true de novo.
+
+        The model was originally designed to run against VCFs produced by GATK from high-throughput sequencing
+        experiments, and uses the GT, AD, DP, GQ, and PL fields. The absence of any of these fields will
+        result in an empty table of results (no called de novo mutations).
+
+        The model looks for de novo events in which both parents are homozygous
+        reference and the proband is a heterozygous. The model makes the simplifying assumption that when this
+        configuration ``x = (AA, AA, AB)`` of calls occurs, exactly one of the following is true:
+        
+            - ``d`` = a de novo mutation occurred in the proband and all calls are true
+            - ``m`` = at least one parental allele is truly non-reference and the proband call is true
+         
+        We can then estimate the posterior probability of a de novo mutation as:
+
+        .. math::
+
+            \\mathrm{P_{\\text{de novo}}} = \\frac{\mathrm{P}(d\,\|\,x)}{\mathrm{P}(d\,\|\,x) + \mathrm{P}(m\,\|\,x)}
+
+        Applying Bayes rule to the numerator and denominator yields
+
+        .. math::
+
+            \\frac{\mathrm{P}(x\,\|\,d)\,\mathrm{P}(d)}{\mathrm{P}(x\,\|\,d)\,\mathrm{P}(d) + \mathrm{P}(x\,\|\,m)\,\mathrm{P}(m)}
```

Comment:
you're getting double pipes. Looks like | should be used directly rather than \|